NetExericise/NetExericise/Array/Exercise03.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NetExericise.Array
+{
+    public class Exercise03
+    {
+        public int OddEvenDifference(int[] arr)
+        {
+            return arr.Sum(i =>
```

Comment:
Đừng đặt tên là `i` cho biến lambda vì `i` thường gợi ý đến `index`, nên đặt là `x`

Thử
```suggestion
            return arr.Sum(x => (1 - 2 * (x % 2)) * i);
```

Reply:
Done